src/internal/log/field.go

```diff
@@ -65,14 +65,17 @@ func RetryAttempt(i int, max int) Field {
 	return zap.Inline(&attempt{i: i, max: max})
 }
 
-// Metadata is a Field that logs the provided metadata (canonicalizing keys, and collapsing
-// single-element values to strings).
+// Metadata is a Field that logs the provided metadata (canonicalizing keys, collapsing
+// single-element values to strings, and removing the Pachyderm auth token).
 func Metadata(name string, md metadata.MD) Field {
 	return zap.Object(name, zapcore.ObjectMarshalerFunc(func(enc zapcore.ObjectEncoder) error {
 		keys := maps.Keys(md)
 		sort.Strings(keys)
 		for _, k := range keys {
 			v := md[k]
+			if k == constants.ContextTokenKey {
+				v = []string{"[MASKED]"}
+			}
```

Comment:
Right now it’s only possible to have a single token, so this is okay. It _might_ be worth iterating through the value, setting each item to `"MASKED"`, in order to preserve an a congruent array, in case zero or multiple tokens someday make it into a request.